messaging.rst

```diff
@@ -137,17 +137,53 @@ Only valid if all the following hold:
+| Field Name            | Field Type           |  Description                                               |
++=======================+======================+============================================================+
+|  command_id           | one byte             | Value 7 indicating ``LockedTransfer``                      |
++-----------------------+----------------------+------------------------------------------------------------+
+|  pad                  | three bytes          | Contents ignored                                           |
++-----------------------+----------------------+------------------------------------------------------------+
+|  nonce                | uint64               | See `Offchain Balance Proof`_                              |
++-----------------------+----------------------+------------------------------------------------------------+
+|  chain_id             | uint256              | See `Offchain Balance Proof`_                              |
++-----------------------+----------------------+------------------------------------------------------------+
+|  message_identifier   | uint64               | An IDfor ``Delivered`` and ``Processed`` acknowledgments   |
++-----------------------+----------------------+------------------------------------------------------------+
+|  payment_identifier   | uint64               | An identifier for the payment that the initiator specifies |
++-----------------------+----------------------+------------------------------------------------------------+
+|  expiration           | uint256              | See `HashTimeLock`_                                        |
++-----------------------+----------------------+------------------------------------------------------------+
+|  token_network_address| address              | See ``token_network_id`` in `Offchain Balance Proof`_      |
++-----------------------+----------------------+------------------------------------------------------------+
+|  token                | address              | Address of the tokoen contract                             |
++-----------------------+----------------------+------------------------------------------------------------+
+|  channel_identifier   | uint256              | See `Offchain Balance Proof`_                              |
++-----------------------+----------------------+------------------------------------------------------------+
+|  recipient            | address              | Destination of this hop of the transfer                    |
++-----------------------+----------------------+------------------------------------------------------------+
+|  target               | address              | Final destination of the payment                           |
++-----------------------+----------------------+------------------------------------------------------------+
+|  initiator            | address              | Initiator of the transfer and person who knows the secret  |
```

Comment:
nitpick: /s/person/party

Reasoning: Machines can also participate in the protocol. We can go into the whole machines are people too argument but maybe let's not :smile: